LayoutTests/resources/ui-helper.js

```diff
@@ -70,6 +70,12 @@ window.UIHelper = class UIHelper {
         await UIHelper.animationFrame();
     }
 
+    static async startMonitoringWheelEvents(args)
```

Comment:
Did you mean to say `...args`? It's strange to call a single argument args.

Reply:
It's only ever a single dictionary argument, but `...args` would work too.